Error during teardown: self = <allure_commons._allure.fixture object at 0x000001AC38A31F30>, args = (), kwargs = {}

    def __call__(self, *args, **kwargs):
        self.parameters = func_parameters(self._fixture_function, *args, **kwargs)
    
        with self:
>           return self._fixture_function(*args, **kwargs)

venv\lib\site-packages\allure_commons\_allure.py:231: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

request = <SubRequest 'allure_attachments' for <Function test_navigation_to_student_page>>

    @pytest.fixture(scope="function", autouse=True)
    def allure_attachments(request):
        """
        Automatically attach screenshots to Allure reports for all tests
        """
        yield  # Run the test
    
        # After each test, attach a screenshot if the page fixture was used
        if "page" in request.fixturenames:
>           page = request.getfixturevalue("page")
E           AssertionError: The fixture value for "page" is not available.  This can happen when the fixture has already been torn down.

conftest.py:27: AssertionError